### DISCUSSION TITLE
Github account has been suspended

### DISCUSSION THREAD

**Discussion by HareemShaikh01** [community/community#138473] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Account user name : Har33m-Sha1kh

Few days ago I noticed something strange happening to my account. Lots of emails had been generated from my account at different open source projects without my knowledge and then my account got suspended. I have filled the Reinstatement form but no response yet please help me  ,how I can restore my account or at least how I can restore my repo codes.

**Comment by LiteBrite82** [comment]
Hi @HareemShaikh01 

Thank you for your patience. The community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. We appreciate your understanding!

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

Thank you!
